wdl2cwl/main.py

```diff
@@ -421,7 +418,10 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:  # type: ignore
             elif len(arguments) == 2:
                 operand, suffix = arguments
                 is_file = isinstance(operand.type, WDL.Type.File)
-                operand = self.get_expr_name(operand.expr)  # type: ignore
+                if isinstance(operand, WDL.Expr.Get):
```

Comment:
The former implementation assumes that the instance for the basename function is always a WDL.Expr.Get expr. This handles the case where the basename function has another function as argument.